### PR TITLE
updating extensions for insiders and stable gallery for April

### PR DESCRIPTION
**Pull request by udeeshagautam**
This PR updates Schema compare, dacpac and SQLProject extensions for insiders and stable gallery for April.

Build for insiders : https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=148998&view=artifacts&pathAsName=false&type=publishedArtifacts

Build for stable: Waiting on release branch

### CODE REVIEW

**Comment by alanrenmsft** [comment]
@udeeshagautam a couple things:
1. could you please update the file name as well? you only updated the version of the extension.
2. looks like the new version of extension requires ADS April release, if so, as per the process, please exclude the changes to stable gallery from this PR and update the rc gallery next week when the release starts.

**Comment by udeeshagautam** [comment]
> @udeeshagautam a couple things:
> 
> 1. could you please update the file name as well? you only updated the version of the extension.
> 2. looks like the new version of extension requires ADS April release, if so, as per the process, please exclude the changes to stable gallery from this PR and update the rc gallery next week when the release starts.

Thanks @alanrenmsft. updated.

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -1230,14 +1230,14 @@
...
-							"lastUpdated": "09/03/2021",
+							"version": "1.10.0",
+							"lastUpdated": "04/08/2022",
 							"assetUri": "",
 							"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 								{
 									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/dacpac/dacpac-1.9.1.vsix"
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/dacpac/dacpac-1.10.1.vsix"
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:1240]: Should be `1.10.0`

**Comment by Charles-Gagnon** [comment]
@udeeshagautam You mentioned updating schema compare but I don't see it in the changes - is that correct?

File: extensionsGallery-insider.json (hunk 1 above)
Comment (Charles-Gagnon) [reply extensionsGallery-insider.json]: Fixed it for you 😄

**Review by alanrenmsft — APPROVED** [review]

**Comment by alanrenmsft** [comment]
@udeeshagautam  gallery updated.

**Comment by udeeshagautam** [comment]
> @udeeshagautam You mentioned updating schema compare but I don't see it in the changes - is that correct?

oh yeah SC was for stable only - removed that file but didn't update the description. 

Thanks both for taking care of this.

**Pull request merged**